src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -8,7 +8,7 @@ import {
     GitOpsOrganisationIdType,
     GitProvider,
 } from './gitops.type'
-import { ProtectedInput } from '../globalConfigurations/GlobalConfiguration'
+import { ProtectedInput, CustomInput as CustomPassInput } from '../globalConfigurations/GlobalConfiguration'
```

Comment:
why are we creating this alias?

Reply:
Because there are importing two CustomInput one from GlobalConfiguration  and one from common.